src/components/ha-selector/ha-selector-text.ts

```diff
@@ -0,0 +1,43 @@
+import { customElement, html, LitElement, property } from "lit-element";
+import { fireEvent } from "../../common/dom/fire_event";
+import { HomeAssistant } from "../../types";
+import "@polymer/paper-input/paper-textarea";
+
+@customElement("ha-selector-text")
+export class HaTextSelector extends LitElement {
+  @property() public hass!: HomeAssistant;
+
+  @property() public value?: any;
+
+  @property() public label?: string;
+
+  protected render() {
+    return html`<paper-textarea
+      .label=${this.label}
+      .value="${this.value}"
+      @keydown=${this._ignoreKeydown}
```

Comment:
Why is this needed?

Reply:
I don't know. It was in the markdown card editor, and I assumed it had to do with the quick select menu perhaps?

Reply:
I think that is for the dialog, but this is not in a dialog, so we can leave it out.

Reply:
Could be. I removed it and don't run into any problems when I test.

But on the other hand, I also removed it from the markdown card editor, and I don't run into any problems there either...

Reply:
press enter in the markdown editor ;-)

Reply:
Yes. That's what I though'd be the problem, and I expected it to close the editor... but it still didn't when I removed the stopPropagation.
Maybe it's a browser dependent thing? I only tried it on macOS Chrome and iOS Safari.

Either way, if that *is* the problem, it should stay in the markdown editor, and be left out here.

Reply:
And that is why we need comments 😄